suite-common/wallet-utils/src/accountUtils.ts

```diff
@@ -588,6 +591,20 @@ export const getTokensFiatBalance = (
     return totalBalance.toFixed();
 };
 
+export const getStakeFiatBalance = (account: Account, coinFiatRate: Rate | undefined) => {
+    if (!coinFiatRate?.rate) return null;
+
+    if (account.networkType !== 'ethereum') return null;
+
+    const { autocompoundBalance } = account?.misc?.stakingPools?.[0] ?? {};
+    if (!autocompoundBalance) return null;
+
+    const balanceInEther = new BigNumber(fromWei(autocompoundBalance, 'ether')).toString();
```

Comment:
Why using `BigNumber` and then still converting to a string? IIRC, `fromWei()` already returns a string.